tests/Integration/Behaviour/Features/Context/Domain/Product/UpdateStatusFeatureContext.php

```diff
@@ -89,6 +91,39 @@ public function updateStatus(bool $status, string $productReference): void
         }
     }
 
+    /**
+     * There are two commands which can change product status.
+     * One is dedicated for only the status change (e.g. used in product list as a toggle),
+     * while another one allows to update status together with other properties (e.g. used in product form).
+     *
+     * @see UpdateProductStatusCommand
+     * @see UpdateProductCommand
+     *
+     * @When /^I (enable|disable) product "(.*)" in a form$/
```

Comment:
added this step because we will have to leave both UpdateStatusCommand and UpdateProductCommand working, so it didnt make sense to follow "update_product" suite implementation as we did in previous unifying PR's. Idk if its the best option, but it works. Wording may be modified, but could find better one instead of `... in a form`

Reply:
Yeah I think it's fine for now to test both commands separately And as we discussed once all is unified we're gonna fuse all the steps related to unified commands into a single one So I guess at that moment we'll remove this one and we'll use the global step to modify some other fields with the status inside (some scenarios involving visibility and status should be interesting since they're both related to indexation)